File: QRCode.Core/Domain/qr_gencode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QRCode.Core.Domain
{
    public class qr_gencode
    {
        public virtual int qrgencodeid { get; set; }
        public virtual string typecode { get; set; }
        public virtual decimal dataid { get; set; }
        public virtual string image { get; set; }
        public virtual string name { get; set; }
        public virtual string code { get; set; }
        public virtual int status { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual int created_by { get; set; }
        public virtual DateTime? lastcreated_date { get; set; }
        public virtual int lastcreated_by { get; set; }
        public virtual int qrpaymentid { get; set; }
    }
}

File: QRCode.FEW/ClientApp/src/app/models/qr_gencode.ts
export interface qr_gencode {
    qrgencodeid: number;
    typecode: string;
    dataid: number;
    image: string;
    name: string;
    code: string;
    status: number;
    created_date: Date;
    created_by: number;
    lastcreated_date: Date;
    lastcreated_by: number;
    qrpaymentid: number;
}
export interface gencodeview {
    qr_name: string;
    qr_img: string;
    qr_tpye: string;
    qr_obj_name: string;
    qr_obj_url: string;
    status_qr: string;
    status_pack: string;
    create_date_qr: string;
    exp_date: string;
    qrgencodeid: number;
    pack_name: string;
    qr_code: string;
    qrpaymentid:number;
}
export interface gencode_status {
    qrgencodeid: number;
    qrpaymentid:number;
    status: number;
    userid:number;
}
export interface count_obj{
    count_enterprise:number;
    count_gencode:number;
    count_payment:number;
    count_product:number;
    count_survey:number;
}